tests/dpsdgym/metrics/pmse.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np

```

Comment:
remote __init__ files in tests. This may be contributed to opendp.whitenoise.synthesizers.metrics at some point. For now they can be here just like evaluator was